carrot/src/main/java/clone/carrot/service/MemberService.java

```diff
@@ -0,0 +1,44 @@
+package clone.carrot.service;
+
+import clone.carrot.doamin.Member;
+import clone.carrot.doamin.dto.MemberCreateDto;
+import clone.carrot.doamin.dto.MemberFindDto;
+import clone.carrot.exception.NotFoundException;
+import clone.carrot.exception.message.ErrorMessage;
+import clone.carrot.repository.MemberRepository;
+import jakarta.persistence.EntityNotFoundException;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+    @Transactional
+    public String createMember(
+            final MemberCreateDto createDto
+            ) {
+        Member member = Member.create(createDto.name(), createDto.age());
+        memberRepository.save(member);
+        return member.getId().toString();
+    }
+
+    public MemberFindDto findMemberById(Long memberId){
```

Comment:
memberId 는 Notnull 일 텐데요, 불변이랑 notnull 을 보장해주면 더 에러가 발생하지 않을 코드가 될것같네요~
```suggestion
    public MemberFindDto findMemberById(final long memberId){
```